### ISSUE TITLE
ffrk-proxy 0.12.0-pre: mv is not recognized as an internal or external command

### ISSUE THREAD

**Issue by andyfeb** [ThauEx/ffrk-proxy#120]
`> ffrk-proxy@0.11.0 start C:\Users\Toro\Documents\Android Projects\Proxy\ffrk-proxy-0.12.0-pre
> babel --plugins inline-json,../babel/config-plugin.js lib/filter/*.js --out-dir public/ && mv public/lib/filter/* public && rm -rf public/lib && node bin/app.js

lib/filter/buddy.js -> public\lib\filter\buddy.js
lib/filter/enemy.js -> public\lib\filter\enemy.js
lib/filter/magicite.js -> public\lib\filter\magicite.js
lib/filter/rounds.js -> public\lib\filter\rounds.js
lib/filter/supporter.js -> public\lib\filter\supporter.js
'mv' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ffrk-proxy@0.11.0 start: `babel --plugins inline-json,../babel/config-plugin.js lib/filter/*.js --out-dir public/ && mv public/lib/filter/* public && rm -rf public/lib && node bin/app.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ffrk-proxy@0.11.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Toro\AppData\Roaming\npm-cache\_logs\2018-04-02T10_01_49_196Z-debug.log`

**Comment by ThauEx** [comment]
I'm aware of this issue. I will try to fix this soon... Can you try to run the `npm start` command in a powershell window? Maybe this works.

**Comment by andyfeb** [comment]
Yeah, I didn't use the bathroom file because my node isn't in the bin. I used 'npm start' and that's what I got.
So, for now, what is the status of the ffrk-proxy? Which version running well on which version of apk?

**Comment by ThauEx** [comment]
The current stable and the pre release are working with the latest APK version.
Are you running `npm start` in CMD or powershell? If CMD, please try in powershell. Otherwise run `node bin/app.js`.

**Comment by ThauEx** [comment]
Try the latest 0.12.0-2 release, I added/modified the cmd script and added a powershell script. You could modify the powershell script and change `.\bin\node.exe` to `node`.
